Doppler.BigQueryMicroservice/Serialization/AllowEmails.cs

```diff
@@ -23,6 +23,6 @@ public AllowedEmails(IReadOnlyList<UserAccessByUser> data)
         }
 
         public List<string> Emails { get; set; }
-
+        public string Languaje { get; set; }
```

Comment:
Mmm, it is arguable, but I do not like to add it as part of the model.

I feel that it is not related to allow or not allow emails.

I tend to think that this configuration could come from another place, some options:
* From the database (user language)
* From the request's query string
* From a request's header

Another option could be using a different name, what do you think about:

```suggestion
        public string NotificationLanguaje { get; set; }
```

Or

```suggestion
        public AllowedEmailsNotification Notification { get; set; }
```

And having another class like this:

```suggestion
        public class AllowedEmailsNotification
        {
            public bool NotifyNewEmails { get; set; }
            public string Language { get; set; }
        }
```

Reply:
Por ahora, vamos a utilizar el lenguaje asociado al usuario (ver campo `idUser` en table `User` y tabla `UserLanguage`)